README.md

```diff
@@ -62,7 +62,13 @@ Our chat channel is to be found on Gitter: https://gitter.im/fossasia/susi_webch
 
 Still having problems? Watch this video to clear your doubts [How to setup SUSI web-chat with surge](https://www.youtube.com/watch?v=vM9cD1pHMDQ&t=240s)
 
+### Deploying on Firebase 
+[Click Here to read the set up guide](https://github.com/arundhati24/chat.susi.ai/blob/firebase_deploy/docs/FIREBASE_DEPLOY.md)
```

Comment:
Please change the link to `.../fossasia/chat.susi.ai...` rather than linking it to your fork. :smile:

Reply:
Oh! That's worth noting :smile: 
But, how can I know the url unless its merged? I worked on my fork of the repository linking the image from the docs folder of my fork.. Could you please suggest what changes should I make? :confused:

Reply:
Change the link to `https://github.com/fossasia/chat.susi.ai/blob/firebase_deploy/docs/FIREBASE_DEPLOY.md` :smile:

Reply:
Would changing the username alone in the URL help?

Reply:
Yes

Reply:
Okay wait

Reply:
Updated the link, it has the branch too, I missed that before. :smile:

Reply:
@anshumanv I updated the links, but now I can't see the image in my fork as well, till it gets merged and I update my fork, because of the changed URL. :confused: